docs/kratos/self-service/flows/verify-email-account-activation.mdx

```diff
@@ -183,408 +134,141 @@ your Identity Schema:
+## Choosing the right strategy
 
-For API Clients, the server typically responds with HTTP 400 Bad Request and the Verification Flow in the response payload as
-JSON.
+Ory supports two strategies for verifying your user's addresses.
 
-#### Successful submission
+```mdx-code-block
+<Tabs>
+  <TabItem value="code" label="One-time codes">
+```
 
-On successful submission, an email will be sent to the provided address:
+When using one-time codes to verify a user's address, Ory Identities sends out an email containing a 6-digit code. The user must
+enter the code in a dedicated UI text field to verify their address.
 
-<CodeTabs items={getFlowMethodLinkSuccess} />
+Alternatively, the email also contains a link to the form that pre-fills the code if the user clicks on it. The user must still
+submit the form for Ory Identities to actually mark the address as verified.
 
-## Unsuccessful verification
+```mdx-code-block
+  </TabItem>
+  <TabItem value="link" label="Magic Links">
+```
 
-If the verification challenge (for example the link in the verification email) is invalid or expired, the user will be HTTP 303
-redirected to the Verification UI.
+When using the "magic link" method, Ory Identities sends the user an email with a "magic link". If the user has access to the
+email associated with the and clicks the link, Ory Identities marks that address as verified.
 
-:::note
+```mdx-code-block
+  </TabItem>
+</Tabs>
+```
 
-When an invalid or expired challenge is used, Ory Identities initializes a new Account Verification flow automatically. This flow
-will always be a Browser-based flow because the challenge is completed by clicking a link!
+If the user completes the verification after the link or code expired, they are asked to re-enter their email address and are sent
+a new email for verification.
 
-:::
+### Comparison
 
-The new Verification Flow includes an error message such as the following:
+"One-time code" (`code`) is the default, preferred, and recommended method by Ory.
 
-<CodeTabs items={getFlowMethodLinkInvalidChallenge} />
+Ory supports "magic links", but considers this method a legacy solution. Currently, the method is supported but should be
+considered deprecated and is set to be removed in future releases.
 
-Please keep in mind that this part of the flow always involves a Browser!
+Consider using the `code` method as it mitigates many of the drawbacks of "magic links":
 
-## Successful verification
+- Some email virus scanners open links in emails to scan them. This invalidates the link and may prevent users from recovering
+  their account even if they have the access to the defined recovery address.
+- Verification flows initialized by apps on mobile phones or smart devices don't work with links.
+- Depending on the device settings, clicking a link from an email can open a different browser than the one used to initialize the
+  verification flow. This can confuse your users.
```

Comment:
I see that this content is both in this doc and in another one in this PR. Is this necessary?

Reply:
It's slightly different.

Reply:
I've extracted it now. PTAL.

Reply:
https://github.com/ory/docs/pull/1120#discussion_r1042337805